src/newrelic.ts

```diff
@@ -1,12 +1,11 @@
-/* Copyright © 2021-2022 Seneca Project Contributors, MIT License. */
+/* Copyright © 2021 Seneca Project Contributors, MIT License. */
 
 
-import NewRelic from 'newrelic'
-
-type NewRelicOptions = {};
-
-type Spec = Record<any, any>
-
+import NewRelic from 'newrelic';
+import { NewRelicOptions, Nullable, Spec } from './types';
+import TracingCollector from './TracingCollector';
+import { Required, Skip, Some } from 'gubu';
+import MetricsCollector from './MetricsCollector';
```

Comment:
convert all file names to lower case, hyphen separate
filenames on windows are case insensitive so you can end up with a repo that "works" on windows but fails on other OS's
also, lower and upper do not sort nicely together